structopt-derive/src/lib.rs

```diff
@@ -11,6 +11,7 @@
 //! for the usage of `#[derive(StructOpt)]`.
 
 #![allow(clippy::large_enum_variant)]
+#![cfg_attr(feature = "external_doc", feature(external_doc))]
```

Comment:
Why would we need that?

Reply:
This is needed so the unstable feature `external_doc` is not unnecessarily required, as it would by itself require nightly rustc. So instead of doing `#![feature(external_doc)]`, I used that other attr instead.

Reply:
I wasn't asking why it's behind `cfg_attr`, I was asking why _structopt_ needs `#[feature(external_doc)]` in the first place :) We don't use `#[doc(include = "...")]` anywhere so we shouldn't need it at all

Reply:
You're right, that unstable feature is always unnecessary. Thanks!